poleno/utils/tests/test_templatetags.py

```diff
@@ -253,3 +256,9 @@ def test_change_lang_tag(self):
             self.assertEqual(r1.content, u'(/en/language/)(/de/language/)(/fr/language/)')
             self.assertEqual(r2.content, u'(/en/language/)(/de/language/)(/fr/language/)')
             self.assertEqual(r3.content, u'(/en/language/)(/de/language/)(/fr/language/)')
+
+        patched = functools.partial(page_not_found, template_name=u'404x.html')
+        with mock.patch(u'django.views.defaults.page_not_found', patched):
+            r4 = self.client.get(u'/language/')
+            self.assertIsInstance(r4, HttpResponseRedirect)
```

Comment:
@martinmacko47 Vsimol som si ze sa pouzivaju dva sposoby. `self.assertIsInstance(obj, class)` a `self.assertIs(type(obj), class)`. Je v tom nejaky rozdiel? Upravil by som to nech sme na vsetkych miestach jednotni.

Reply:
@viliambalaz Rozdiel je v tom, ze `self.assertIs(type(obj), class)` oveduje, ze `obj` je instancia `class`, nie vsak jej subclassov. Ale `self.assertIsInstance(obj, class)` overuje, ze `obj` je instancia `class` alebo jej subclassov.

Kde teda chceme overit, ze nieco je presne dana classa, treba pouzit `assertIs`. Ak chceme overit, ze nieco je dana classa alebo jej podlasy, treba pouzit `assertIsInstance`